monitor.py

```diff
@@ -859,23 +859,14 @@ class WorkReaper(ReaperMonitor):
     """
     MODEL_CLASS = Work
 
-    def __init__(self, *args, **kwargs):
-        from external_search import ExternalSearchIndex
-        search_index_client = kwargs.pop('search_index_client', None)
-        super(WorkReaper, self).__init__(*args, **kwargs)
-        self.search_index_client = (
-            search_index_client or ExternalSearchIndex(self._db)
-        )
-
```

Comment:
Removing this code will work but, my experience is that instantiating an `ExternalSearchIndex` is pretty expensive. Please reinstate this; otherwise the reaper will be really slow.

Reply:
This is added back in now.